src/ObjCRuntime/Protocol.cs

```diff
@@ -34,14 +38,21 @@ public Protocol (IntPtr handle)
 			this.handle = handle;
```

Comment:
Do we want to keep this one for .NET?

Reply:
IIRC I tried to remove it, ran into a problem, and had to put it back in. However, I can't remember exactly the scenario and I might be getting confused with something else, so I've pushed this change to see what CI says about it.